src/components/App/ConfigEditor.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-form
+    ref="form"
+    v-model="validated"
+  >
+    <v-container>
+      <v-row justify="center">
+        <v-col class="py-0">
+          <v-text-field
+            v-model="globalConfig.numDaysBack"
+            type="number"
+            label="Days back from today to take"
+            :rules="[required, positive]"
+            @change="changed = true"
+          />
+        </v-col>
+        <v-col
+          cols="auto"
+          class="py-0"
+        >
+          <v-checkbox
+            v-model="globalConfig.showBrowser"
+            label="Show browser"
+            @change="changed = true"
+          />
+        </v-col>
+      </v-row>
+    </v-container>
+
+    <v-btn
+      color="primary"
+      :disabled="!readyToSave"
+      @click="submitForm()"
+    >
+      Save
+    </v-btn>
+  </v-form>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters, mapActions } from 'vuex';
+import { GLOBAL_CONFIG_GETTER, UPDATE_GLOBAL_CONFIG_ACTION } from '@/store/modules/Config';
+import { VForm } from '@/types/vuetify';
```

Comment:
Workaround for `Vue.$refs` that hold an abstract `Vue` instances so you have to convert it to the specific type.